05-html-css/01-the-web.md

```diff
@@ -0,0 +1,74 @@
+# 🌐 Intro to the web 🌐
+
+
+## 📚 Learning Goals 📚
```

Comment:
Is this an appropriate place to also talk about how the Internet actually works? I like this video: https://www.youtube.com/watch?v=PBWhzz_Gn10 and then an activity that I showed Kari a couple cohorts ago where you use notecards/pieces of paper to send messages around the room and simulate lost packets, packet #s, etc.  ....too late to add this in? @PilgrimMemoirs @kariabancroft @CheezItMan